src/node.cc

```diff
@@ -397,6 +397,12 @@ MaybeLocal<Value> StartExecution(Environment* env, const char* main_script_id) {
           ->GetFunction(env->context())
           .ToLocalChecked()};
 
+  InternalCallbackScope callback_scope(
+    env,
+    Local<Object>(),
```

Comment:
```suggestion
    Object::New(env()->isolate()),
```

The `Local<Object>()` form breaks `executionAsyncResource` tracking stuff as it doesn't produce a valid object.